File: its/projects/CSharpVBNetCoverage/CSharpConsoleApp.Test/UnitTest1.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace CSharpConsoleApp.Test
{
    [TestClass]
    public class UnitTest1
    {
        [TestMethod]
        public void TestMethod1()
        {
            Assert.AreEqual(10, Program.Passthrough(10));
        }
    }
}

File: its/projects/CSharpVBNetCoverage/VBNetConsoleApp.Test/UnitTest1.vb
Imports Microsoft.VisualStudio.TestTools.UnitTesting

Namespace VBNetConsoleApp.Test
    <TestClass>
    Public Class UnitTest1
        <TestMethod>
        Sub TestSub()
            Assert.AreEqual(10, Module1.Passthrough(10))
        End Sub
    End Class
End Namespace

